source/client-side-encryption/tests/countDocuments.json

```diff
@@ -1,252 +1,257 @@
 {
-    "runOn": [
-        {
-            "minServerVersion": "4.1.10"
+  "runOn": [
```

Comment:
NOTE: the real changes in this file much smaller. I've just updated lines 204, 211, 215-221. Not sure why github highlighted a lot of lines below